main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/30x/enrober/pkg/server"
+
+	"k8s.io/kubernetes/pkg/client/restclient"
+)
+
+func main() {
+
+	//Default to local client
+	clientConfig := restclient.Config{
+		Host: "127.0.0.1:8080",
+	}
+
+	envState := os.Getenv("DEPLOY_STATE")
+	switch envState {
+	case "PROD":
```

Comment:
Not sure I understand the need for 2 cases here.  Shouldn't this just be an env variable that is set?